### DISCUSSION TITLE
Problem installing

### DISCUSSION THREAD

**Discussion by MuslimSage** [bmaltais/kohya_ss#663] [Q&A]
I go through the installation, when I launch gui.bat I get this...

[notice] To update, run: python.exe -m pip install --upgrade pip
PS C:\ai\Kohya\kohya_ss\kohya_ss> ./gui
System Information:
System: Windows, Release: 10, Version: 10.0.19044, Machine: AMD64, Processor: AMD64 Family 23 Model 113 Stepping 0, AuthenticAMD

Python Information:
Version: 3.11.2, Implementation: CPython, Compiler: MSC v.1934 64 bit (AMD64)

Virtual Environment Information:
Path: C:\ai\Kohya\kohya_ss\kohya_ss\venv

GPU Information:
Name: NVIDIA GeForce RTX 3070, VRAM: 8192 MiB

Validating that requirements are satisfied.
Error: The following packages are missing:
 - accelerate==0.15.0
 - albumentations==1.3.0
 - altair==4.2.2
 - bitsandbytes==0.35.0
 - dadaptation==1.5
 - diffusers[torch]==0.10.2
 - easygui==0.98.3
 - einops==0.6.0
 - ftfy==6.1.1
 - gradio==3.27.0; sys_platform != 'darwin'
 - lion-pytorch==0.0.6
 - opencv-python==4.7.0.68
 - pytorch-lightning==1.9.0
 - safetensors==0.2.6
 - tensorboard==2.10.1 ; sys_platform != 'darwin'
 - tk==0.1.0
 - toml==0.10.2
 - transformers==4.26.0
 - voluptuous==0.13.1
 - fairscale==0.4.13
 - timm==0.6.12
 - huggingface-hub==0.13.3; sys_platform != 'darwin'
 - tensorflow==2.10.1; sys_platform != 'darwin'
 - lycoris_lora==0.1.4

Run ←[33mupgrade.ps1←[0m or ←[33mpip install -U -r requirements.txt←[0m to resolve the missing requirements listed above...

**Comment by vkage** [comment]
run pip install --r requirements.txt
in venv
